### PR TITLE
Fix MySqlCluster script init

### PR DESCRIPTION
**Pull request by neykov**
`CREATION_SCRIPT_{URL, CONTENT} `was applied to the master node pre-replication-init so there was no way to initialize the slave nodes. Improved by:
  * Applying `CREATION_SCRIPT_{URL, CONTENT} ` on all nodes, including slaves pre-replication-init
  * Introducing a new `MASTER_CREATION_SCRIPT_{URL, CONTENT} ` config applied on master only post-replication-init

### CODE REVIEW

**Comment by neykov** [comment]
Build failure not related, forced a new one.

File: software/database/src/test/java/org/apache/brooklyn/entity/database/mysql/MySqlClusterIntegrationTest.java [hunk 1]
```diff
@@ -40,4 +42,22 @@ public void test_localhost() throws Exception {
...
+                String runDir = member.getAttribute(MySqlNode.RUN_DIR);
+                if (runDir != null) {
+                    Os.deleteRecursively(runDir);
+                }
+            }
+        }
+    }
+
+    private Location getLocation() {
+        return mgmt.getLocationRegistry().resolve("vagrant", ImmutableMap.of("minRam", "2G"));
```
Comment (aledsage) [inline software/database/src/test/java/org/apache/brooklyn/entity/database/mysql/MySqlClusterIntegrationTest.java]: Needs comment to say what config is required in brooklyn.properties for this to work. Presumably it won't work for other people, or for jenkins?

**Comment by aledsage** [comment]
Looks good; just one comment about vagrant location.

File: software/database/src/test/java/org/apache/brooklyn/entity/database/mysql/MySqlClusterIntegrationTest.java (hunk 1 above)
Comment (neykov) [inline software/database/src/test/java/org/apache/brooklyn/entity/database/mysql/MySqlClusterIntegrationTest.java]: This wasn't supposed to get committed. Replaced with `localhost`.

**Comment by sjcorbett** [comment]
@neykov Thanks. Will merge.

**Pull request merged**